node/sc/bridge_manager.go

```diff
@@ -520,14 +505,14 @@ func (bm *BridgeManager) LogBridgeStatus() {
 	logger.Info("VT : Service -> Main Chain", "request", s2mTotalRequestNonce, "handle", s2mTotalHandleNonce, "pending", s2mTotalRequestNonce-s2mTotalHandleNonce)
 }
 
-// SubscribeTokenReceived registers a subscription of TokenReceivedEvent.
-func (bm *BridgeManager) SubscribeTokenReceived(ch chan<- RequestValueTransferEvent) event.Subscription {
-	return bm.scope.Track(bm.tokenReceived.Subscribe(ch))
+// SubscribeRequestEvent registers a subscription of TokenReceivedEvent.
```

Comment:
How about change 'TokenReceivedEvent' in this comment to requestEventFeeder?

Reply:
reflected :)